### ISSUE TITLE
Using a link along with a sidekick results in indefinite "Networking" state

### ISSUE THREAD

**Issue by tsilenzio** [rancher/rancher#2040]
Whenever I use a sidekick along with a link, the PostgreSQL container will always be in an indefinite "Networking" state.

```
data:
    image: tsilenzio/base
    stdin_open: true
    tty: true
    command:
        - cat

postgresql:
    image: tsilenzio/postgresql

phpfpm:
    image: tsilenzio/phpfpm
    labels:
        io.rancher.sidekicks: data
    links:
        - postgresql
```

If I remove the ```postgresql``` link or the ```data``` sidekick label, then it will work as expected

Rancher Server: v0.37.0
Rancher Agent: v0.8.2
Rancher Agent Instance: v0.4.1
Rancher Compose: v0.2.6

**Comment by tsilenzio** [comment]
If it helps any, I am testing this on a machine where the rancher server and rancher agent are on the same machine.

I am also using Docker v1.8.2

**Comment by tsilenzio** [comment]
In the meantime I have set the ```net``` to ```bridge``` for all containers, and this has solved the problem, but I would prefer to user rancher's managed network

**Comment by sangeethah** [comment]
@tsilenzio ,this issue is fixed with rancher/server:v0.37.1

I tried to launch the service you have provided in the issue using rancher-compose and see both "phpfpm" and "postgresql" get to active state.

**Comment by tsilenzio** [comment]
Perfect, thank you very much!
